include/rta/fullsupernodelist.h

```diff
@@ -126,6 +126,9 @@ class FullSupernodeList
     size_t refreshedItems() const;
 
 
+    static std::string printAuthSample(const std::vector<SupernodePtr> &authSample);
```

Comment:
why don't you use `operator<<` instead??

Reply:
replaced with operator<<